### DISCUSSION TITLE
Product Variant Shippable?

### DISCUSSION THREAD

**Discussion by simon-online** [saleor/saleor#7909] [Q&A]
### What I'm trying to achieve
Currently there is a "Is this product shippable?" field defined on a product type.

I'm looking at creating a product that has a digital format and physical format by adding a digital product variant and a physical product variant to the product. Which means if a customer checks out the digital product variant then the product isn't shippable but if a customer checks out the physical product variant then the product is shippable.

This will make product entry simpler as you don't have to create duplicate products ie. one product / product type for the digital non-shippable format and another product / product type for the physical shippable format.

### Describe a proposed solution
Would it be possible to add a "Is this product shippable?" field to a product variant so that this can be defined as an override at the product variant level?

Alternatively I could add a variant attribute for "Product shippable?" however I'm not sure if this would be a completely successful workaround. Would it be possible to override the product shippable status using a product variant attribute during the checkout process or would this not be a good workaround?

**Comment by patrys** [comment]
We do not plan to support mixing product types within one product. You will need to use product attributes to link to alternative formats or wait for bundle products to be implemented (as those would allow you to create a product page that allows the user to choose between different products) but be aware that bundles are not on our immediate roadmap.
